### PR TITLE
Uikit conversion v1

### PR DESCRIPTION
**Pull request by lloan**
Items updated:
- Mobile navigation
- Desktop navigation 
- Mobile and desktop navigation share single source of truth 
- Removed v1 components 
- Sponsor section tweaked for mobile 
- Added Link component to links 
- Added breadcrumb to blog pages 
- Removed randomization of success stories, converted to slider (need indicator that its a slider - issue?) 
- Tweaked footer for mobile 
- Moved blog post utility functions to own file. 
- Removed data folder
- Removed abort from singleEvent
- Converted nav logo in to a link back home
- Added linter plugin for react hooks

### CODE REVIEW

File: pages/post.js [hunk 1]
```diff
@@ -51,16 +68,49 @@ function Post(props) {
         if (loading) return <div>Loading</div>;
```
Comment (lloan) [inline pages/post.js]: Wonder if we can add a loader component here instead of text?

File: sass/components/layout/_sponsors.scss [hunk 2]
```diff
@@ -12,4 +13,12 @@
     margin: 20px auto;
     display: block;
   }
+
+  .uk-grid-small div {
+
```
Comment (lloan) [inline sass/components/layout/_sponsors.scss]: We should think about adding a coding style spec. for sass.

Comment (lloan) [reply sass/components/layout/_sponsors.scss]: @tonynguyen111997

**Comment by VelazquezLuis** [comment]
I think we should toggle uk-open off when clicking on a link in the nav menu in mobile version

**Comment by VelazquezLuis** [comment]
success stories should have a uk-slider-nav  list to know it changes.

**Pull request merged**